components/apimgt/org.wso2.carbon.apimgt.rest.api.commons/src/main/java/org/wso2/carbon/apimgt/rest/api/common/interceptors/RESTAPISecurityInterceptor.java

```diff
@@ -105,6 +105,7 @@ public boolean preCall(Request request, Response response, ServiceMethodInfo ser
                 return false;
             }
         } else if (requestURI.contains("/store")) {
+            /*TODO: Remove below swagger.json serving path when old Store UUF app is removed ~tmkb*/
```

Comment:
Mmm I don't think we need to remove this. There might be clients who want to read the swagger in json format as well. But as per the offline discussion we had, the converted json has some issues so we need to fix them.